packages/components/src/components/date-picker/date-picker.tsx

```diff
@@ -0,0 +1,48 @@
+import { Component, Prop, h, Element, Event, EventEmitter } from '@stencil/core';
+
+@Component({
+  tag: 'ifx-date-picker',
+  styleUrl: 'date-picker.scss',
+  shadow: true,
+})
+
+export class DatePicker {
+  @Element() el: HTMLElement;
+  @Prop() size: string = 's';
+  @Prop() error: boolean = false;
+  @Prop() success: boolean = false;
+  @Prop() disabled: boolean = false;
+  @Event() ifxDate: EventEmitter;
+
+  getDate(e) { 
```

Comment:
maybe we should add some form of input validation

Reply:
Validating what? Since I am integrating the input type=date, the browser does the date validation.